### DISCUSSION TITLE
OpeTopography cannot be reached

### DISCUSSION THREAD

**Discussion by SeppeVE** [domlysz/BlenderGIS#639] [Q&A]
Hi, I was just checking GIS out and tried to follow the steps but when i went to "Get elevation" it gave a error saying OpenTopography can't be reached. So i tried to just skip this since its a flat area and do the "Get OSM" but that didn't work either. I'm on the newest version of blender as well as on the V2.2.5 of GIS. Is this a general bug or something on my end?

**Comment by ik0jre** [comment]
Hi everybody.

I tried blendergis and can say it's very useful plugin.

But from January 1st, 2022 OpenTopography changed rest method. this the note: 
"NOTE: Starting on January 1st, 2022, an API authorization key will be required for this API. Users can request an API key via [myOpenTopo](https://portal.opentopography.org/myopentopo) in the OpenTopography portal. See this [blog post](https://opentopography.org/blog/introducing-api-keys-access-opentopography-global-datasets) for more information."
Available at web page https://opentopography.org/developers.

Could be an blendergis update soon ?
Thak you so much for everything have you done.

Giuseppe

**Comment by ik0jre** [comment]
NOTE just tried:
After read a post from relector-tuxnix I appended '&API_Key=my_key' to the end of the connection string under the addons preferences and it worked just fine.

How to get a APY key you must read note at:
https://opentopography.org/developers

**Comment by domlysz** [comment]
Fixed https://github.com/domlysz/BlenderGIS/commit/72cbb0507b80c2116fd4252b9082b6beba170a89
